README.md

```diff
@@ -11,7 +11,8 @@ Preparing your python virtual environment:
 
     $ virtualenv venv
     $ source venv/bin/activate
-    $ pip install -r requirements.txt
+    $ pip install -e .  # To create an editable install of this package
+    $ pip install git+https://github.com/ManageIQ/manageiq-api-client-python.git  # Or you can install it from git directly
```

Comment:
@mfalesni I think it's not clear that it's an option.

Since this is a getting started, I think we should just leave this (no need for comments):

    $ pip install -e .

And then have at the bottom a new paragraph saying something like:

> If you want to install from git directly you can run:
> `pip install git+https://github.com/ManageIQ/manageiq-api-client-python.git`